src/pyscipopt/scip.pxd

```diff
@@ -348,7 +358,8 @@ cdef extern from "scip/scip.h":
         pass
 
     ctypedef struct SCIP_BRANCHRULE:
-        pass
+        char* name
+        SCIP_RETCODE (*branchexeclp)(SCIP* scip, SCIP_BRANCHRULE* branchrule, SCIP_Bool allowaddcons, SCIP_RESULT* result)
```

Comment:
Please remove the callback.

Reply:
I think we do actually need this. We have branch rules written in Python that internally call `branchexeclp` for existing branch rules (e.g. `relpscost`).

Reply:
Hmm, I understand but I feel very mixed about adding this to PySCIPOpt. At least for `relpscost` you don't need this because you can use `SCIPexecRelpscostBranching` from `branch_relpscost.h`. Would this be enough for your application? If yes, then please remove the callback from the struct. Thanks!

@fserra and @mattmilten, what do you generally think about adding the callback?